nova-peru/nova.github.io
# Feat/resend

**Pull Request Description:**

### Feature: Resend and Toast Notifications for Contact Form

**Changes:**
- Implemented message resend capability in the contact form.
- Added toast notifications for user feedback.

**Objective:**
Enhances user experience by allowing message resending and providing real-time feedback through toast notifications in the contact form.

**Testing:**
1. Visit the contact form.
2. Submit a message.
3. Test the resend feature.
4. Observe toast notifications for instant feedback.
